### ISSUE TITLE
Continuing to use v1.23.0 due to better spacebar line seeking implementation.

### ISSUE THREAD

**Issue by Shpantze** [Julow/Unexpected-Keyboard#540]
Hey how's it going, I've had some issues with upgrades since 1.23 where the spacebar doesn't seem to allow moving back and forth through lines anymore in Acode (android app)... version 1.23 does work fine and I've been sticking with it, which is cool for me, but just thought I'd mention it for future versions.

**Comment by sdrapha** [comment]
I also like the old behavior better and I accomplish that with a custom layout

only including here the piece of code for the space key
```xml
<key width="4.6" key0="space" key5="left" key6="right" key7="switch_numeric" key8="switch_backward" slider= "true"/> 
```
*ignore the other extra keys, my layout is all the way unique from the default

With that, Acode will work just fine.

**Comment by sdrapha** [comment]
https://github.com/Julow/Unexpected-Keyboard/assets/25059996/0a121afb-1a83-40ad-b2ad-d852c152c58b

**Comment by Spike-from-NH** [comment]
I'm baffled. `slider="true"` is the key, but doesn't it function by generating `left` and `right` keystrokes as you move your finger horizontally? In fact, `slider` would seem to override `key5` and `key6`.

You're saying Acode won't move from the end of one line to the start of the next line (nor back) with `left` and `right` keystrokes?

**Comment by Julow** [comment]
I think this is a bug in Acode. Related issue: https://github.com/deadlyjack/Acode/issues/907

A workaround is implemented in 2c52e94. Debug build: https://github.com/Julow/Unexpected-Keyboard/actions/runs/7770064675

**Comment by sdrapha** [comment]
Just to clarify my custom layout approach. The difference from the default layout is not the `slider` portion.
The default layout has `key5="cursor_left"` then I changed it to `key5="left"` that is the older behavior with no new enhanced cursor navigating features.

**Comment by Shpantze** [comment]
> I think this is a bug in Acode. Related issue: https://github.com/deadlyjack/Acode/issues/907
> 
> A workaround is implemented in 2c52e94. Debug build: https://github.com/Julow/Unexpected-Keyboard/actions/runs/7770064675

Great, thank you for your response!
